autoscheduler/scraper/views.py

```diff
@@ -112,3 +114,20 @@ def get(self, request):
         data = Grades.objects.instructor_performance(subject, course_num,
                                                      instructor, honors)
         return Response(data)
+
+@api_view(['GET'])
+def get_last_updated(request):
+    """ Takes in a term and attempts to retrieve when that term was last updated.
+        Defaults to 11/1/20 if it does not find one.
+    """
+    term = request.query_params.get('term')
+
+    if not term:
+        return Response(status=400)
+
+    try:
+        last_updated = Term.objects.get(code=term).last_updated
+    except Term.DoesNotExist:
+        last_updated = datetime(2020, 11, 1)

```

Comment:
This should return undefined so it doesn't actually show anything on the frontend since the current output is misleading